spectral_cube/tests/test_spectral_cube.py

```diff
@@ -74,18 +75,31 @@ def test_huge_disallowed():
 
     cube, data = cube_and_raw('vda_Jybeam_lower.fits')
 
-    data = np.empty([1e2,1e3,1e3])
     cube = SpectralCube(data=data, wcs=cube.wcs)
 
-    assert cube._is_huge
+    assert not cube._is_huge
+    
+    # We need to reduce the memory threshold rather than use a large cube to
+    # make sure we don't use too much memory during testing.
```

Comment:
Sounds good - I've added an issue here: https://github.com/radio-astro-tools/spectral-cube/issues/262 - maybe we want a ``--large-cube-tests`` option to enable those tests